contextproject-ms3/src/main/java/contextproject/models/Track.java

```diff
@@ -29,7 +31,10 @@
    * @param abPath
    *          Path of the mp3 file
    */
-  public Track(String abPath) {
+  public Track(String abPath, Hashtable<String, String> info) {
```

Comment:
Could you leave the old Track constructor (with only abPath) as well? (and then let them both call the same method). Then everything will be nice and backwards compatible.